UI/login.html

```diff
@@ -6,7 +6,7 @@
         <link rel="stylesheet" type="text/css" href="css/login.css" media="screen">
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
     </head>
-    <body>
+    <body style="background: url(image/email.png); background-size:cover">
```

Comment:
I do think the background shouts kinda a lot to the extend of almost making the page content absorbed by it. This kinda interferes with UX.

Reply:
okay

Reply:
Your UI looks great to me, only that the background image seems to be too graphical; so I suggest you make it more faint to reduce graphical interference or use a different background image/color/texture.
It's great that you have already implemented some input validation on your signup form. keep it up!!

Reply:
okay, am also looking into gradients